api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+        enrichment and partitioning. The steps performed on reception of incoming message are:
+
+        1. Every validation rule specified in the `EventType` will be checked in order against the incoming
+        Events. Validation rules are evaluated in the order they are defined and the Event is **rejected**
+        in the first case of failure with a status of 400 Bad request. If the offending validation rule
+        provides information about the violation it will be included in the Problem response.
+        If the `EventType` defines schema validation it will be performed at this moment.
+        **TBD behaviour in case of violation within a stream of events**)
+
+        1. Once the validation succeeded, the content of the Event is updated according to the enrichment
+        rules in the order the rules are defined in the `EventType`.
+        No preexisting value might be changed (even if added by an enrichment rule). Violations on this will
+        force the immediate **rejection** of the Event with a 409 Conflict status. The invalid overwrite
+        attempt will be included in the respose's Problem object.
+
+        1. The incoming Event's relative ordering is evaluated according to the rule on the
+        `EventType`. Failure to evaluate the rule will **reject** the Event.
+        **Restriction**: POSTing Events based on its `EventType` requires it to have a
+        `PartitioningStrategy` defined. Violating this condition will produce a 405 Not allowed.
+
+      parameters:
+        - name: name
+          in: path
+          type: string
+          description: Name of the EventType
+          required: true
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+          format: flow-id
+      responses:
+        '200':
+          description: Ok
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: EventType not found.
+          schema:
+              $ref: '#/definitions/Problem'
+        '405':
+          description: Not allowed.
+          schema:
+              $ref: '#/definitions/Problem'
+        '409':
+          description: Conflict.
+          schema:
+              $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /topics:
+    get:
+      description: Lists all known `Topics` of this Nakadi cluster
```

Comment:
Basically corresponds to GET /EventTypes?fields=topic i.e. filter topic field from event types.
Why not doing like this -- instead of inventing new resource (see also comment below)?